tests/mesh_velocity_output.prm

```diff
@@ -0,0 +1,116 @@
+# Test for relaxin topography of the free surface after an 
+# initial sinusoidal perturbation
```

Comment:
There's a typo in the first line. But the test is about something different anyway.